guides/common/modules/ref_expiring-rhel-client-support-in-rhel-lifecycle-status.adoc

```diff
@@ -0,0 +1,20 @@
+[id="expiring_rhel_client_support_in_rhel_lifecycle_status_{context}"]
+= Expiring {EL} Client Support in RHEL Lifecycle Status
+
+{Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {EL} clients.
+
+When either the end of maintenance support or the end of extended lifecycle support approaches in a year, you will see a notification banner in the {ProjectWebUI} if you have hosts in that {EL} version.
+The notification message provides information about the {EL} version, the number of hosts running that version in your environment, the lifecycle support, and the expiration date.
+Along with other information, the {EL} lifecycle column is visible in the report.
+For any hosts that are not running {EL}, *Unknown* will appear in the *Lifecycle Status* and *Report* column.
```

Comment:
```suggestion
For any hosts that are not running {EL}, *Unknown* displays in the *RHEL Lifecycle status* and *Last report* columns.
```
Please double check this suggestion with SME but those are the names of columns that I see on my testing instance.